x/delegation/keeper/genesis.go

```diff
@@ -44,7 +45,7 @@ func (k Keeper) InitGenesis(
 					// they are the LzNonce and TxHash
 				}
 				if err := k.delegateTo(ctx, delegationParams, false); err != nil {
-					panic(err)
+					panic(errorsmod.Wrap(err, "failed to delegate to operator"))
```

Comment:
The `InitGenesis` function signature is fixed. I cannot alter it. 

Since this is a problematic error - one that is observed during genesis - it should panic.